dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -80,6 +80,13 @@ def show
     authorize! :read, ScriptLevel
     @script = ScriptLevelsController.get_script(request)
 
+    # @view_as_user is used to determine redirect path for bubble choice levels
+    @view_as_user = if params[:user_id] && params[:user_id] != current_user.id
```

Comment:
NIT: Replace with ternary operator
`@view_as_user = params[:user_id] && params[:user_id] != current_user.id ? User.find(params[:user_id]) : current_user`

Reply:
i tried that first but the line is too long and our rubocop doesn't allow multiline ternary expressions: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MultilineTernaryOperator